frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -307,11 +267,38 @@ const ManagePolicyPage = ({
       enabled: isRouteOk && isPremiumTier && !!teamIdForApi,
       onSuccess: (data) => {
         setTeamPolicies(data.policies);
-        setInheritedPolicies(data.inherited_policies);
       },
     }
   );
 
+  const {
+    data: teamPoliciesCountIncludeInherited,
+    isFetching: isFetchingTeamCountIncludeInherited,
+    refetch: refetchTeamPoliciesCountIncludeInherited,
+  } = useQuery<
+    IPoliciesCountResponse,
+    Error,
+    number,
+    ITeamPoliciesCountQueryKey[]
+  >(
+    [
+      {
+        scope: "teamPoliciesCountIncludeInherited",
+        query: searchQuery,
```

Comment:
> Search doesn't seem to be working
@jacobshandling 
Search should work once the API is implemented, since it's server side